pkg/upgrade/upgrade_test.go

```diff
@@ -0,0 +1,44 @@
+package upgrade
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/spf13/cobra"
+)
+
+func TestReporter(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "reporter testing")
```

Comment:
@Mischulee Could you update the text here to correctly identify the test suite. Thanks!

Reply:
Sorry I havent noticed, fixed :sweat_smile: